prisma/schema.sql

```diff
@@ -35,265 +48,371 @@ CREATE TYPE PhysicianStatus as ENUM('ACTIVE', 'INACTIVE');
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
 -- Create email verification requests table
 CREATE TABLE verification_requests (
-  id            SERIAL PRIMARY KEY NOT NULL,
-  identifier    VARCHAR(255) NOT NULL,
-  token         VARCHAR(255) NOT NULL UNIQUE,
-  expires       TIMESTAMPTZ NOT NULL,
-  created_at    TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at    TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+  id SERIAL PRIMARY KEY,
+  identifier VARCHAR(255) NOT NULL,
+  token VARCHAR(255) NOT NULL UNIQUE,
+  expires TIMESTAMPTZ NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
 -- Create physicians table
 CREATE TABLE physicians (
-  id              SERIAL PRIMARY KEY NOT NULL,
-  name            VARCHAR(255) NOT NULL,
-  msp_number      INTEGER UNIQUE NOT NULL,
-  address_line_1  VARCHAR(255) NOT NULL,
-  address_line_2  VARCHAR(255),
-  city            VARCHAR(255) NOT NULL,
-  province        Province NOT NULL DEFAULT 'BC',
-  postal_code     CHAR(6) NOT NULL,
-  phone           VARCHAR(50) NOT NULL,
-  status          PhysicianStatus NOT NULL DEFAULT 'ACTIVE',
-  notes           TEXT,
-  created_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+  msp_number INTEGER UNIQUE NOT NULL,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  phone VARCHAR(50) NOT NULL,
+  address_line_1 VARCHAR(255) NOT NULL,
+  address_line_2 VARCHAR(255),
+  city VARCHAR(255) NOT NULL,
+  province Province NOT NULL DEFAULT 'BC',
+  country VARCHAR(255) NOT NULL DEFAULT 'Canada',
+  postal_code CHAR(6) NOT NULL,
+  status PhysicianStatus NOT NULL DEFAULT 'ACTIVE',
+  notes TEXT,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+  PRIMARY KEY(msp_number)
 );
 
 -- Create guardians table
 CREATE TABLE guardians (
-  id              SERIAL PRIMARY KEY NOT NULL,
-  first_name      VARCHAR(255) NOT NULL,
-  middle_name     VARCHAR(255),
-  last_name       VARCHAR(255) NOT NULL,
-  address_line_1  VARCHAR(255) NOT NULL,
-  address_line_2  VARCHAR(255),
-  city            VARCHAR(255) NOT NULL,
-  province        Province NOT NULL,
-  postal_code     CHAR(6) NOT NULL,
-  phone           VARCHAR(50) NOT NULL,
-  relationship    VARCHAR(255) NOT NULL,
-  notes           TEXT,
-  created_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at      TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+  id INTEGER,
+  first_name VARCHAR(255) NOT NULL,
+  middle_name VARCHAR(255),
+  last_name VARCHAR(255) NOT NULL,
+  phone VARCHAR(50) NOT NULL,
+  address_line_1 VARCHAR(255) NOT NULL,
+  address_line_2 VARCHAR(255),
+  city VARCHAR(255) NOT NULL,
+  province Province NOT NULL DEFAULT 'BC',
+  country VARCHAR(255) NOT NULL DEFAULT 'Canada',
```

Comment:
A concern about including the `country` field (across all tables): although it seems like a valid scenario for people to be outside of Canada, then the `province` field wouldn't cover the alternate country and we don't allow people to enter a non-canadian province equivalent, and the 6 character requirement on postal code would also be invalid (american zip codes are 5 digits i think)

Reply:
Great catch! Currently, we are assuming that all applicants, physicians, and guardians have addresses in BC, Canada - there is no UI for editing these fields, only for viewing. I agree though, we should add some flexibility to support the possibility of international addresses in the future. I will change the type of `postal_code` to `VARCHAR(255)`. Province can remain as an enum - it is easier to add support for more enum values than to change the column data type